File: src/main/kotlin/a/BearAndBigBrother.kt
package a

/* Problem Link: https://codeforces.com/problemset/problem/791/A */

fun main() {
    var (a, b) = readln().split(" ").map { it.toInt() }
    var years = 0

    while (a <= b) {
        years++
        a *= 3
        b *= 2
    }

    print(years)
}

File: Codeforces/C++/problemset/Bear and Big Brother.cpp
// Problem Link: https://codeforces.com/problemset/problem/791/A

#include <iostream>
using namespace std;
int main()
{
    long long a,b,y = 0;
    cin >> a >> b;
    while(a <= b)
    {
        a *= 3;
        b *= 2;
        y++;
    }
    cout << y;
    return 0;
}
